config/config.play.js

```diff
@@ -8,16 +9,13 @@ module.exports = {
   },
   dist: 'dist-play',
   postcss: config.postcss,
-  templateCompiler: true,
-  webpack: {
-    resolve: {
-      modules: [
-        path.resolve('src')
-      ]
-    }
+  webpack (config) {
+    ['app', 'preview'].forEach(entry => {
```

Comment:
I hope you should add `hmrEntry` option to `poi` due to write path with hard coded.